bots/pr/src/main/java/org/openjdk/skara/bots/pr/LimitedCensusInstance.java

```diff
@@ -55,27 +64,43 @@ private static Namespace namespace(Census census, String hostNamespace) {
         if (namespace == null) {
             throw new RuntimeException("Namespace not found in census: " + hostNamespace);
         }
-
         return namespace;
     }
 
-    private static Optional<JCheckConfiguration> configuration(HostedRepositoryPool hostedRepositoryPool, HostedRepository remoteRepo, String name, String ref) throws IOException {
-        return hostedRepositoryPool.lines(remoteRepo, Path.of(name), ref).map(JCheckConfiguration::parse);
+    private static Optional<JCheckConfiguration> configuration(HostedRepository remoteRepo, String name, String ref) {
+        Optional<List<String>> conf;
+        try {
+            conf = Optional.of(Arrays.stream(remoteRepo.fileContents(name, ref).split("\n")).toList());
+        } catch (RuntimeException e) {
+            if (e.getMessage().contains("Request returned bad status:")) {
+                if (!e.getMessage().contains("404")) {
+                    throw e;
+                }
+            }
+            conf = Optional.empty();
+        }
+        return conf.map(JCheckConfiguration::parse);
```

Comment:
First, when we try to read the `.jcheck/conf` from the remote repo, maybe the request will throw `uncheckedRestException`. Only when the status code is `404`, conf will be set to `Optional.empty()`, so that it will be treated as `MissingJcheckConfException` later.